src/python/WMCore/MicroService/MSOutput/MSOutput.py

```diff
@@ -581,10 +583,10 @@ def msOutputProducer(self, requestRecords):
         #    To generate the object from within the Function scope see above.
         counter = 0
         for request in requestRecords:
-            counter += 1
             pipeLineName = msPipeline.getPipelineName()
             try:
                 msPipeline.run(request)
+                counter += 1
```

Comment:
Likewise here. Previous behavior was counting number of requests "produced". With this change, it is number of requests "successfully produced"

Reply:
in my latest commit i restored the position where this counter is incremented, and improved its name to `counterOk`, because it is counting only for wfs that are properly processed.